components/automate-ui/src/app/components/tabs/tabs.component.html

```diff
@@ -1,8 +1,11 @@
 <ul class="nav nav-tabs-list">
   <li *ngFor="let tab of tabs" class="nav-tabs-item">
     <button (click)="selectTab(tab)" class="nav-tab" [class.active]="tab.active">
-      {{tab.tabTitle}}
+      {{tab.tabTitle}} 
     </button>
+    <span *ngIf="tab.tabTitle === 'Policyfiles'" class="beta-tag"><span>Beta</span></span>
+    <span *ngIf="tab.tabTitle === 'Details'" class="beta-tag"><span>Beta</span></span>
+    <span *ngIf="tab.tabTitle === 'Reset Admin Key'" class="beta-tag"><span>Beta</span></span>
```

Comment:
Instead of hard-coding these in the template, consider defining a constant array in the backfile code...
```
const betaTabs = ['Policyfiles', 'Details', 'Reset Admin Key'];
```
... and then in here just do this (also should not need the nested `span`):
```suggestion
    <span *ngIf="betaTabs.includes(tab.tabTitle)" class="beta-tag">Beta</span>
```